specs/AnimatedVisualPlayer Spec.md

```diff
@@ -0,0 +1,145 @@
+AnimatedVisualPlayer Spec
+===
+
+# Background
+
+Today, developers can create animated visuals to use in their UI by using the
+[AnimatedVisualPlayer](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.AnimatedVisualPlayer)
+control.
+`AnimatedVisualPlayer` can play any
+[IAnimatedVisual](https://docs.microsoft.com/uwp/api/Microsoft.UI.Xaml.Controls.IAnimatedVisual)
+implementation,
+but the typical way to get an `IAnimatedVisual` implementation is to use
+the LottieGen tool to generate code from a JSON file that was created from Adobe AfterEffects.
+
+[More information on Lottie and AnimatedVisualPlayer](https://docs.microsoft.com/en-us/windows/communitytoolkit/animations/lottie)
+
+There are some performance issues with the current implementation of LottieGen and AnimatedVisualPlayer: 
+1)	Complex Lottie animations can instantiate a lot of composition animations 
+which may impact performance and overall battery life. 
+2)	Lottie files that have a lot of complex animations even when they are paused
+have some cost to performance. 
```

Comment:
```suggestion
2)	Lottie files that have a lot of complex animations
have some cost to performance even when they are paused. 
```

The point is not that Lottie files can have complex animations even when paused. It's that Lottie files with complex animations have performance costs even when paused.

Reply:
that is a cleaner way to communicate the complexity! Committing suggestion.